test/transform-feature.test.js

```diff
@@ -0,0 +1,59 @@
+const { test, snapshot } = require("node:test");
+const { transformSync } = require("../dist/index.js");
+const path = require("node:path");
+
+snapshot.setResolveSnapshotPath((testPath) => {
+	return path.join(
+		__dirname,
+		"snapshots",
+		`${path.basename(testPath)}.snapshot`,
+	);
+});
+
+test("should transform TypeScript class fields", (t) => {
+	const inputCode = `
+    class Counter {
+      count: number = 0;
+      increment() {
+        this.count++;
+      }
+    }
+  `;
+	const { code } = transformSync(inputCode, {
+		mode: "transform",
+		sourceMaps: true,
+	});
+	t.assert.snapshot(code);
+});
+
+test("should transform TypeScript private class fields", (t) => {
+	const inputCode = `
+    class Counter {
+      #count: number = 0;
+      increment() {
+        this.#count++;
+      }
+      getCount(): number {
+        return this.#count;
+      }
+    }
+  `;
+	const { code } = transformSync(inputCode, {
+		mode: "transform",
+		sourceMaps: true,
+	});
+	t.assert.snapshot(code);
```

Comment:
why no execution test here with vm?

Reply:
I wonder if there is a problem if I add all the tests on this page

Reply:
its ok

Reply:
ı added, thanks